v2.2/architecture/life-of-a-distributed-transaction.md

```diff
@@ -142,6 +134,11 @@ If an operation encounters a write intent for a key, it attempts to "resolve" th
   If the push succeeds, the operation continues.
 
   If this push fails (which is the majority of the time), this transaction goes into the [`TxnWaitQueue`](https://www.cockroachlabs.com/docs/stable/architecture/transaction-layer.html#txnwaitqueue) on this node. The incoming transaction can only continue once the blocking transaction completes (i.e., commits or aborts).
+- _Missing_, the resolver checks the write intent's timestamp. If it was created within the last two seconds, it exhibits the `PENDING` behavior, with the addition of tracking the push in the range's timestamp cache, which will inform the transaction that its timestamp was pushed once the transaction record gets created.
```

Comment:
s/checks/consults/

s/it exhibits/it treats the transaction record as exhibiting/